charts/kserve-resources/templates/configmap.yaml

```
@@ -415,6 +415,20 @@ data:
          "defaultDeploymentMode": "Serverless"
        }
 
+     # ====================================== SERVICE CONFIGURATION ======================================
+     # Example
+     service: |-
+       {
+         "serviceClusterIPNone": true
+       }
+     service: |-
+       {
+          # ServiceClusterIPNone is a flag to indicate if the service should have a clusterIP set to None.
+          # This is particularly useful for headless services to allow users to use out of the box balancers.
+          # If the DeploymentMode is Raw, the default value for ServiceClusterIPNone is true
```

```suggestion
          # ServiceClusterIPNone indicates if KServe should create  headless services (clusterIP=None).
          # If the DeploymentMode is Raw, the default value for ServiceClusterIPNone is true
```

External LBs is probably just one use case for a headless service. I'd omit such details.

---

I see, okay